recipes/binutils/binutils.inc

```diff
@@ -12,12 +12,12 @@ BUGTRACKER = "http://sourceware.org/bugzilla/"
 
 RECIPE_TYPES = "cross sdk-cross machine canadian-cross"
 
-inherit autotools make-vpath gettext
+inherit autotools make-vpath
 
 require conf/fetch/gnu.conf
 SRC_URI = "${GNU_MIRROR}/binutils/binutils-${PV}.tar.bz2 "
 
-DEPENDS = "native:flex native:bison"
+DEPENDS = "native:flex native:bison native:makeinfo native:m4"
```

Comment:
Yes.  Maybe we should only add it to default.conf and all.conf instead then.
That is probably both the right thing, and will allow us to build test this PR without the texinfo recipe merged in OE-lite/base.